iree/turbine/kernel/wave/templates/paged_decode_attention.py

```diff
@@ -272,8 +273,10 @@ def loop(
         res = res_mm * reciprocal_sum
         res_max_log_sum = res_max + tkw.log2(res_sum)
 
-        tkw.write(res_max_log_sum, output_max, elements_per_thread=1)
-        tkw.write(res, output, elements_per_thread=STORE_ELEMS_PER_THREAD)
```

Comment:
Could you also move the operations before the write and after the for loop into the conditional?

Reply:
Just want to make sure that also works, since we wouldn't want to do that work if count is 0.

Reply:
done